Documentation/check-build.sh

```diff
@@ -52,7 +52,11 @@ describe_spelling_errors() {
     find "${spelldir}" -type f -print0 | xargs -0 sed 's/^/* Documentation\//'
 
     # Print a hint on how to add new correct words to the list of good words
-    new_words="$(sed -E "s/^([^:]+:){2} \(([^ ]+)\).*/\2/g" "${spelldir}"/* | sort -u | tr '\r\n' ' ' | sed "s/'/\\\\\\\\'/g")"
+    new_words="$(find "${spelldir}" -type f -print0 | \
+        xargs -0 sed -E "s/^([^:]+:){2} \(([^ ]+)\).*/\2/g" | \
```

Comment:
nit, is `xargs -0` cross-platform? I think we ended up needing to rip this out of `install/kubernetes` because it was causing problems for developers.

Reply:
Good question, no idea about the portability.

But then is anyone running `check-build.sh` outside of a Docker container? The Makefile builds and uses the cilium/docs-builder image to run it on `make html` or `make live-preview`, and we don't document running the script directly anywhere. So this should be fine?

Reply:
Makes sense yep :+1: Also I just realized that a few lines up, `xargs -0` is already present in this script.